c7n/utils.py

```diff
@@ -386,6 +386,27 @@ def __contains__(self, other):
         return super(IPv4Network, self).__contains__(other)
 
 
+    # Redefine from ipaddress backport to address that this is not
+    # in the python 3.6 stdlib implementation.
+    # Note this is copyright 2007 Google, 2007 and licensed to the PSF
+    # and released under the python license.
+    # https://docs.python.org/3/license.html
+
```

Comment:
Too many blank lines lint error

Reply:
putting in review comments for changes for something ci catches seems a bit silly.